webpack.config.js

```diff
@@ -38,6 +38,10 @@ module.exports = {
     http: 'empty',
   },
 
+  externals: {
+    xpath: 'xpath',
```

Comment:
This is a good idea, to exclude it in the bundling.

Reply:
thank you